.flake8

```diff
@@ -17,4 +17,4 @@ max-line-length = 119
 copyright-check = True
 select = E,F,W,C
 copyright-regexp=Copyright \(c\) Facebook, Inc. and its affiliates. All Rights Reserved
-ignore=W503,E203
+ignore=W503,E203,F407
```

Comment:
Can you use a `# noqa` on the specific offending line?
See [flake](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html) docs.

Reply:
done!